rts/at/sra/README.md

```diff
@@ -0,0 +1,68 @@
+OVERVIEW
+--------
+An example of how to use REST-AT service.
```

Comment:
What is a REST-AT service?

Reply:
Hi @mmusgrov based on [review](https://github.com/jbosstm/quickstart/pull/434#discussion_r1190171503) I removed SRA and used on REST-AT.

Reply:
You only changed this one example of using the term SRA, what about all of the other references to SRA in the PR - I don't really want to call out all usages of SRA because the PR should be consistent in how it uses terminology.

Reply:
> Hi @mmusgrov based on [review](https://github.com/jbosstm/quickstart/pull/434#discussion_r1190171503) I removed SRA and used on REST-AT.

When you respond to a review comment by pushing an update please can you address all comments made in the PR, either via PR updates or via conversation responses. If you keep pushing changes incrementally then it is difficult for me to know what I should be looking for and it is very time consuming to keep up with your incremental pushes,